src/main/java/by/exadel/internship/tool/ScheduleJob.java

```diff
@@ -0,0 +1,26 @@
+package by.exadel.internship.tool;
+
+import by.exadel.internship.repository.location.CountryRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.context.annotation.Profile;
+import org.springframework.scheduling.annotation.EnableScheduling;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Component;
+
+import java.util.UUID;
+
+@Component
+@EnableScheduling
+@Profile("development")
```

Comment:
Looks like profile here should be something like heroku...

Reply:
https://github.com/viktarkolbik/backend_team_four/pull/40#discussion_r622775505

this same Pofile, i gues

Reply:
It was just an example:

For example, branch development is deployed on dev environment with appropriate configuration.
So please change name
application-development.yaml or application-dev.yaml

Reply:
i renamed this to heroku